bedrock/firefox/templates/firefox/developer-quantum-whatsnew.html

```diff
@@ -0,0 +1,240 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/shared" %}
+
+{% extends "firefox/base-pebbles.html" %}
+
+{% block page_title %}{{_('Firefox Developer Edition')}}{% endblock %}
+{% block page_desc %}{{ _('Firefox Developer Edition is the blazing fast browser that offers cutting edge developer tools and latest features like CSS Grid support and framework debugging') }}{% endblock %}
+{% block page_image %}{{ static('img/firefox/products/developer/quantum/page-image.png') }}{% endblock %}
+{% block page_favicon %}{{ static('img/firefox/products/developer/quantum/favicon.png') }}{% endblock %}
+{% block page_favicon_large %}{{ static('img/firefox/products/developer/quantum/favicon-196.png') }}{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_developer_quantum_whatsnew' %}
+{% endblock %}
+
+{% block body_id %}firefox-developer{% endblock %}
+{% block body_class %}firefox-developer{% endblock %}
+
+{% block content %}
+<main role="main">
+  <section class="section section-intro">
+    <div class="content">
+      <div class="section-body">
+        <h1 class="page-title">{{ _('Congrats. You now have Firefox Quantum: Developer Edition.') }}</h1>
+        <p class="tagline">{{ _('This isn’t just an update. This is Firefox Quantum: A brand new Firefox that has been rebuilt from the ground-up to be faster, sleeker, and more powerful than ever.') }}</p>
+
+        <ul class="features-list gallery-list has-three-cols">
+          <li>
+            <img class="icon" src="{{ static('img/firefox/products/developer/quantum/devtools-icon.svg') }}" alt="">
+            <h3>{{ _('Firefox Devtools') }}</h3>
+            <p class="cta"><a href="#devtools" class="button button-hollow">{{ _('Learn more') }}</a></p>
```

Comment:
Should we use smooth scrolling for these?

Reply:
`scroll-behavior: smooth;` CSS FTW 👍